src/CMakeLists.txt

```diff
@@ -251,7 +250,13 @@ if(BUILD_SHARED_LIBS)
   install(FILES ${VALHALLA_SOURCE_DIR}/COPYING ${VALHALLA_SOURCE_DIR}/CHANGELOG.md
     DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/doc/libvalhalla0"
     COMPONENT shared)
-else()
```

Comment:
This whole `else` clause was confusing to me: why create so-symlink if `BUILD_SHARED_LIBS=OFF` at all?
```
if(UNIX)
    add_custom_target(${LIBVALHALLA_SO_LINK} ALL
...
```

Reply:
good point :smile:

Reply:
I'm glad I'm not alone with the confusion ;P